src/vector/vdrawable.h

```diff
@@ -54,6 +55,11 @@ class VDrawable {
     void preprocess(const VRect &clip);
     void applyDashOp();
     VRle rle();
+    void setName(const char *name)
+    {
+        if (name) mName = strdup(name);
```

Comment:
Guess this keypath is unique id, does it necessary copy names every frames?

Reply:
Here there is no frame info. The name value is set when the item's constructor is called.